src/components/Sidebar/Sidebar.jsx

```diff
@@ -29,9 +42,14 @@ function Sidebar() {
     checkIsAdmin();
   });
 
+  useEffect(() => {
+    getBusinessName();
+  },
+  [backend]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'getBusinessName'. Either include it or remove the dependency array.